default.nix

```diff
@@ -29,6 +34,7 @@ let
     shellToolOverrides = ghc: super: {
       cabal-install = pkgs.haskellPackages.cabal-install;
       ghcid = pkgs.haskellPackages.ghcid;
+      spectacle = pkgs.spectacle;
```

Comment:
What required this package?  (I couldn't find anything with https://packdeps.haskellers.com/reverse)

Reply:
The doc build in `build-docs.sh`